src/components/ebay-progress-spinner/README.md

```diff
@@ -0,0 +1,12 @@
+# ebay-progress-spinner
+
+```marko
+<ebay-progress-spinner large />
+```
+
+## Attributes
+
+Name | Type | Stateful | Required | Description
+--- | --- | ---- | --- | ---
+`large` | Boolean | No | No | If true, large spinner is displayed; else, normal
```

Comment:
I vote for making this an enumerated type rather than a boolean. For example, size="large". With "small" being the default size. Reason being we may add other sizes in future (extra-small, extra-large, etc).